utils/sysctl/sysctl.go

```diff
@@ -0,0 +1,49 @@
+// Ref: https://github.com/kubernetes/kubernetes/blob/cb2ea4bf7c029e595f44ee62013c982626fb5bd4/staging/src/k8s.io/component-helpers/node/utils/sysctl/sysctl.go
+
+package sysctl
+
+import (
+	"io/ioutil"
+	"path"
+	"strconv"
+	"strings"
+)
+
+const (
+	sysctlBase = "/proc/sys"
+)
+
+// Interface is an injectable interface for running sysctl commands.
+type Interface interface {
+	// GetSysctl returns the value for the specified sysctl setting
+	GetSysctl(sysctl string) (int, error)
```

Comment:
why not Get and Set instead?

Reply:
No reason, I was just copying what was originally done. Changing to `Get` and `Set` now